includes/class-apple-news.php

```diff
@@ -79,6 +79,32 @@ public static function get_filename( $path ) {
 	}
 
 	/**
+	 * Validate settings and see if any updates need to be performed.
+	 *
+	 * @param array $wp_settings An array of settings loaded from WP options.
+	 *
+	 * @access public
+	 * @return array The modified settings array.
+	 */
+	public function validate_settings( $wp_settings ) {
+		// If this option doesn't exist, either the site has never installed
+		// this plugin or they may be using an old version with individual
+		// options. To be safe, attempt to migrate values. This will happen only
+		// once.
+		if ( false === $wp_settings ) {
```

Comment:
older versions would potentially pass this arg as `false`? The param above states this should be an array, just wanted to double check that this isn't supposed to be an `empty()` or `null` check instead.